wml_test_schedule

```diff
@@ -166,6 +166,8 @@
 0 event_test_filter_attack_specials
 0 event_test_filter_attack_on_moveto
 0 event_test_filter_attack_opponent_weapon_condition
+0 event_test_filter_attack_opponent_weapon_condition_no_triggered
+0 event_test_filter_attack_student_weapon_condition
```

Comment:
To fit [Ravana's request about showing warnings](https://github.com/wesnoth/wesnoth/pull/9202#issuecomment-2284690424), all six of the new tests ought to return status 9 (BROKE STRICT(PASS)). See also the comments in #9202 about deduplicating the warnings.

Reply:
i don't show warning here

Reply:
But Ravana (and Pentarctagon) have said that they want a warning, so it needs to be added. It doesn't need to give exactly the same text as #9202, but it needs to identify either the unit, the special or the filter well enough for UMC authors to work out what it's referring to, and thus give them enough information to fix their code.

Reply:
i don't undetstand, when i test in game, message don't show, it is normal?

Reply:
I only added them in the logs (or stdout / stderr on Linux). It might be a good improvement  to show the warning in the in-game chat area to, but we'll probably have more feedback from Ravana about it.

Reply:
added for the log but i perfer not show that ingame